From f6dd9a1be5de38e3ac0b75d1018d9f723bfe8b10 Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Wed, 1 Mar 2017 17:57:56 +0800
Subject: [PATCH] PM / devfreq: rk3399_dmc: rename of_get_opp_table

The function doesn't get something from dts, it is more appropriate to
rename of_get_opp_table to rk3399_dmcfreq_init_freq_table.

Change-Id: I8c4994d45ff4d0654d034483e091bbb225a1ea61
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 drivers/devfreq/rk3399_dmc.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/devfreq/rk3399_dmc.c b/drivers/devfreq/rk3399_dmc.c
index 1e25c3441155..98312f9b0d62 100644
--- a/drivers/devfreq/rk3399_dmc.c
+++ b/drivers/devfreq/rk3399_dmc.c
@@ -337,8 +337,8 @@ static struct dram_timing *of_get_ddr_timings(struct device *dev,
 	return timing;
 }
 
-static int of_get_opp_table(struct device *dev,
-			    struct devfreq_dev_profile *devp)
+static int rk3399_dmcfreq_init_freq_table(struct device *dev,
+					  struct devfreq_dev_profile *devp)
 {
 	int count;
 	int i = 0;
@@ -448,7 +448,7 @@ static int rk3399_dmcfreq_probe(struct platform_device *pdev)
 		return -EINVAL;
 	}
 
-	if (of_get_opp_table(dev, devp))
+	if (rk3399_dmcfreq_init_freq_table(dev, devp))
 		return -EFAULT;
 
 	of_property_read_u32(np, "upthreshold",
-- 
2.35.3

